src/enc/get_params.cc

```diff
@@ -1,15 +1,19 @@
 #include "get_params.h"
 
 using namespace v8;
-using namespace brotli;
 
-BrotliParams getParams(Local<Object> userParams) {
-  BrotliParams params;
+BrotliEncoderParams getParams(Local<Object> userParams) {
+  BrotliEncoderParams params = {
+    BROTLI_MODE_GENERIC,
```

Comment:
Shouldn't this be `BROTLI_DEFAULT_MODE` instead?
https://github.com/google/brotli/blob/master/enc/encode.h#L35